### ISSUE TITLE
Versions newer than 1.5.0 refusing to open

### ISSUE THREAD

**Issue by dceightfive** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#153]
Hi,

I am having trouble opening any version of the app that is newer than 1.5.0, I believe it is to do with the Material Skin as 1.6.0 was when it was introduced. 

I'm trying to run it on Windows 7 Enterprise Edition x64, and I have got I get the following prerequisites installed; Visual Studio C++ Redist 2012 (x86), Visual Studio C++ Redist 2012 (x64), and the latest version of flash, and I have also updated my graphics drivers to the latest available.

When trying to launch the app it just shows the orange window border with a white background for a short amount of time then it crashes out.

This is what I see in my debug log:
<code>
[0122/111437:ERROR:renderer_main.cc(200)] Running without renderer sandbox
[0122/111540:ERROR:renderer_main.cc(200)] Running without renderer sandbox
[0122/111540:WARNING:backend_impl.cc(1570)] Messed up entry found.
[0122/111540:WARNING:backend_impl.cc(1778)] Destroying invalid entry.
</code>

Hopefully this is enough information to figure out as to why it is not working, let me know if you need any more.
If anyone has any clue as to why it is failing to open, it would be much appreciated.

Thanks,
Dean

**Comment by MarshallOfSound** [comment]
You need VS C++ Redist 2013

Those errors aren't actually errors.  Check Windows Event Manager for errors

**Comment by dceightfive** [comment]
Thanks for your help MarshallOfSound.

I do already have VS C++ Redist 2013 installed both x86 and x64, along with 2005, 2008, 2010, so I'm not sure that, that is the problem.

Looking in the Windows Event Viewer I see an application error of:
```
Faulting application name: Google Play Music.exe, version: 1.0.0.0, time stamp: 0x564f036a
Faulting module name: KERNELBASE.dll, version: 6.1.7601.17932, time stamp: 0x50327672
Exception code: 0xe0434352
Fault offset: 0x0000c41f
Faulting process id: 0x2160
Faulting application start time: 0x01d15841148f87f1
Faulting application path: C:\Program Files (x86)\GPMDP\Google Play Music.exe
Faulting module path: C:\Windows\syswow64\KERNELBASE.dll
```

and a .NET runtime error:
```
Application: Google Play Music.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.Net.WebException
Stack:
   at System.Net.WebClient.DownloadDataInternal(System.Uri, System.Net.WebRequest ByRef)
   at System.Net.WebClient.DownloadString(System.Uri)
   at System.Net.WebClient.DownloadString(System.String)
   at Google_Play_Music.GPMResouceHandlerFactory.GetResourceHandler(CefSharp.IWebBrowser, CefSharp.IBrowser, CefSharp.IFrame, CefSharp.IRequest)
   at <Module>.CefSharp.Internals.ClientAdapter.GetResourceHandler(CefSharp.Internals.ClientAdapter*, CefRefPtr<CefResourceHandler>*, CefRefPtr<CefBrowser>*, CefRefPtr<CefFrame>*, CefRefPtr<CefRequest>*)
```

Is there something else I am missing?

Thanks, 
Dean

**Comment by MarshallOfSound** [comment]
Are you using a proxy?  Or a VPN?

**Comment by gdaddy184** [comment]
I am having the same issue. It was working fine for me until this morning.

**Comment by MarshallOfSound** [comment]
@gdaddy184 

> Are you using a proxy? Or a VPN?

**Comment by dceightfive** [comment]
Thanks again Samuel,

No, not using a proxy or VPN, as I stated, the latest version works fine on my laptop but not on the PC which are both using the same connection. 

Can you think of anything else that may be causing the crash?

**Comment by MarshallOfSound** [comment]
Well it's something to do with one of the webrequests I do.  It is probably something  like firewall rules, or network config.   I'll make a debug build for you tonight and we can try see whats going on

**Comment by MarshallOfSound** [comment]
This has been tagged with `Fixed in 3.0.0` and in preparation for the release of `3.0.0` these issues are being automatically closed.

Get excited for **3.0.0** :cactus: :cake: :tada: :balloon: 

> Sam Bot
